lib/main_ltp.pm

```diff
@@ -18,31 +18,51 @@
 package main_ltp;
 use base 'Exporter';
 use Exporter;
-use testapi qw(check_var get_var);
+use testapi qw(check_var get_var set_var);
 use autotest;
 use Archive::Tar;
 use utils;
 use LTP::TestInfo 'testinfo';
 use File::Basename 'basename';
-use main_common qw(load_bootloader_s390x boot_hdd_image get_ltp_tag);
+use version_utils qw(is_released is_sle);
+use main_common qw(boot_hdd_image get_ltp_tag load_ayinst_tests load_boot_tests load_bootloader_s390x load_inst_tests load_reboot_tests loadtest);
 use 5.018;
 use Utils::Backends 'is_pvm';
 # FIXME: Delete the "## no critic (Strict)" line and uncomment "use warnings;"
 # use warnings;
 
 our @EXPORT = qw(
+  load_baremetal_tests
   load_kernel_tests
   loadtest_from_runtest_file
 );
 
-sub loadtest {
+sub loadtest_kernel {
     my ($test, %args) = @_;
     autotest::loadtest("tests/kernel/$test.pm", %args);
 }
 
+sub load_baremetal_tests {
```

Comment:
I wonder why you would want this in main_ltp.pm - this is nothing ltp related?

Reply:
OK, should I move it into `main_common.pm`? The point is to move it from `products/sle/main.pm` to be reachable also by `products/opensuse/main.pm`. LTP tests (and maybe other kernel tests) use the same logic on o3 and osd. Sharing code removes duplicity.

Reply:
main_common would be a more reasonable place. I added this for sle only, as we do not have the possibility to use our hardware also for openSUSE.

Reply:
I'll put it into main_common, but the this part is code used for any test which requires installation via yast (we use it for sonic & tails, but it's not specific to it):

```
load_boot_tests();                                              
get_var("AUTOYAST") ? load_ayinst_tests() : load_inst_tests();  
load_reboot_tests();
```
And when we dig into the details, also "baremetal" tests name can be confusing because we used it for baremetal tests in kernel & network (which aren't the only bare metal tests).